packages/bundle/src/adaptiveCards/Attachment/AdaptiveCardHacks/private/useAdaptiveCardModEffect.ts

```diff
@@ -0,0 +1,93 @@
+import { useCallback, useEffect, useMemo, useRef } from 'react';
+
+import useLazyRef from './useLazyRef';
+import useValueRef from './useValueRef';
+
+import type { AdaptiveCard } from 'adaptivecards';
+
+type ModFunction<TArgs extends unknown[] = []> = (
+  adaptiveCard: AdaptiveCard,
+  cardElement: HTMLElement,
+  ...args: TArgs
+) => () => void;
+
+class Mod<TArgs extends unknown[]> {
+  constructor(mod: ModFunction<TArgs>) {
```

Comment:
I think you can do.. Not sure
```suggestion
  constructor(private #mod: ModFunction<TArgs>) {}
```

Reply:
![image](https://user-images.githubusercontent.com/1622400/179900804-cae0083a-5226-4ff7-8ee3-5963e70a810d.png)

Seems only working for TS's fake private modifier, but not JS's real `#` modifier.